application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.4" reference="Analytics.Code.AnalyticsTranslations" locale="">
+  <web>Analytics.Code</web>
+  <name>AnalyticsTranslations</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>AnalyticsTranslations</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>plain/1.0</syntaxId>
+  <hidden>true</hidden>
+  <content># Class fields
+Analytics.Code.AnalyticsDashboardClass_startDate=Start Date
+Analytics.Code.AnalyticsDashboardClass_endDate=End Date
+Analytics.Code.AnalyticsDashboardClass_description=Description
+Analytics.Code.AnalyticsDashboardClass_displayedStatistics=displayed Statistics
+Analytics.Code.AnalyticsDashboardClass_startDate.hint=Start time of the date interval used on all statistics of this dashboard.
+Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interval used on all statistics of this dashboard.
+Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
+Analytics.Code.AnalyticsDashboardClass_displayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
+
+
+
```

Comment:
Just for consistency, if you should leave the same amount of spaces between categories, i.e. here you have 3, and below there is just 1. Just 1 new line should be enough, to diminish the need of scrolling.